1.9/usage/cli/command-reference.md

```diff
@@ -582,49 +582,62 @@ By default, `dcos node ssh` connects to the private IP of the node, which is onl
 
 ## dcos task
 
-    Description:
-        Manage DC/OS tasks.
-    
-    Usage:
-        dcos task --info
-        dcos task [--completed --json <task>]
-        dcos task log [--completed --follow --lines=N] <task> [<file>]
-        dcos task ls [--long] <task> [<path>]
-    
-    Command:
-        log
-            Print the task log. By default, the 10 most recent task logs from stdout
-            are printed.
-        ls
-            Print the list of files in the Mesos task sandbox.
-    
-    
-    Options:
-        --completed
-            Print completed and in-progress tasks.
-        -h, --help
-            Print usage.
-        --info
-            Print a short description of this subcommand.
-        --follow
-            Dynamically update the log.
-        --json
-            Print JSON-formatted list of tasks.
-        --lines=N
-            Print the last N lines. The default is 10 lines.
-        --long
-            Print full Mesos sandbox file attributes.
-        --version
-            Print version information.
-    
-    Positional Arguments:
-        <file>
-            Specify the sandbox file to print. The default is stdout.
-        <path>
-            The Mesos sandbox directory path. The default is '.'.
-        <task>
-            A full task ID, a partial task ID, or a regular expression.
-    
+```
+Description:
+    Manage DC/OS tasks.
+
+Usage:
+    dcos task --help
+    dcos task --info
+    dcos task exec [--interactive --tty] <task> <cmd> [<args>...]
```

Comment:
Is this merged and available in the current CLI? I thought that the PR was not yet merged. https://github.com/dcos/dcos-cli/pull/838